docs/configuration.md

```diff
@@ -312,7 +313,7 @@ In some cases the host value is not present in an event (or an IP address is in
 who require a true hostname be attached to each event, SC4S provides an optional facility to perform a reverse IP to
 name lookup. If the variable `SC4S_USE_REVERSE_DNS` is set to "yes", SC4S
 will first check `host.csv` and replace the value of `host` with the value specified that matches the incoming IP address.
```

Comment:
first checks `host.csv` and replaces the value of `host` with the specified value that matches the incoming IP address.